terraform/iam_module/dbx_workspace_create/variables.tf

```diff
@@ -1,4 +1,4 @@
-variable "name_postfix" {
+variable "workspace_env" {
   description = "String to be appended at the end of resource name properties."
```

Comment:
Skal vi klargjøre hvilke verdier dette kan være? `String to be appended ... [dev/test/prod]`